compose.yaml

```
@@ -52,6 +52,7 @@ services:
       target: online
     ports:
       - "80:8000"
```

`80` is the default port for HTTP. Maybe let's use another available one like `5003`

```suggestion
      - "5003:8000"
```